interface/main/tabs/main.php

```diff
@@ -121,7 +122,7 @@
     const isSms = "<?php echo !empty($GLOBALS['oefax_enable_sms'] ?? null); ?>";
     const isFax = "<?php echo !empty($GLOBALS['oefax_enable_fax']) ?? null?>";
     const isServicesOther = (isSms || isFax);
-    const allowTelemetry = <?php echo js_escape($allowTelemetry); ?>;
+    var telemetryEnabled = <?php echo js_escape(TelemetryService::isTelemetryEnabled()); ?>;
```

Comment:
@sjpadgett , for some reason I needed to have this set as a var for it to be seen by tabs_view_model.js script. There are some comments above about how let/const may be out of scope when try to call from other places. This also allows it to be modified (ie. would be nice to turn it on when opt-in rather than needing to logout/login; this is very optional since only gonna gain a little bit of data, but have the option to add this).

Reply:
I put that note in back when I did the jsGlobals!